packages/wasm-utils/src/note/mod.rs

```diff
@@ -24,13 +28,138 @@ macro_rules! console_log {
 	($($t:tt)*) => (crate::types::log(&format_args!($($t)*).to_string()))
 }
 
+enum JsUtxoInner {
+	Bn254(Utxo<FrBn254>),
+}
+
+#[wasm_bindgen]
+pub struct JsUtxo {
+	#[wasm_bindgen(skip)]
+	pub inner: JsUtxoInner,
+}
+impl JsUtxo {
+	pub fn new_from_bn254_UTXO(utxo: Utxo<FrBn254>) -> Self {
+		Self {
+			inner: JsUtxoInner::Bn254(utxo),
+		}
+	}
+
+	pub fn new_from_Bls381_UTXO(_utxo: Utxo<FrBn254>) -> Self {
```

Comment:
Please remove bls381 for now, also since you're passing in a Bn254 utxo which is not correct.